exporter/templates_test.go

```diff
@@ -835,14 +905,14 @@ func printXmlString(xmlString string) {
 
 func xmlRootNodeForQrdaOid(qrdaOid string) *xml.ElementNode {
 	fileName := "_" + qrdaOid + ".xml"
-	printXmlString(generateXML(fileName, getDataForQrdaOid(qrdaOid)))
+	//printXmlString(generateXML(fileName, getDataForQrdaOid(qrdaOid)))
 	return xmlRootNode(generateXML(fileName, getDataForQrdaOid(qrdaOid)))
 }
 
 // same as xmlRootNodeForQrdaOid() function but allows custom input data (should be an EntryInfo struct)
 func xmlRootNodeForQrdaOidWithData(qrdaOid string, data interface{}) *xml.ElementNode {
 	fileName := "_" + qrdaOid + ".xml"
-	printXmlString(generateXML(fileName, data))
+	//printXmlString(generateXML(fileName, data))
```

Comment:
Is there a reason this is commented instead of removed? otherwise I'm good to merge this in.